cmd/kubeadm/app/cmd/init.go

```diff
@@ -234,7 +234,8 @@ func (i *Init) Run(out io.Writer) error {
 	}
 
 	// PHASE 3: Bootstrap the control plane
-	if err := controlplanephase.WriteStaticPodManifests(i.cfg); err != nil {
+	manifestPath := filepath.Join(kubeadmconstants.KubernetesDir, kubeadmconstants.ManifestsSubDirName)
```

Comment:
FYI, https://github.com/kubernetes/kubernetes/pull/50173 will hopefully merge later today and will make this easier